tests/fixtures/identity.sql

```diff
@@ -171,7 +171,8 @@ SPLIT('new york', '.')
 X((y AS z)).1
 X(a.b = 1)
 (x AS y, y AS z)
-REPLACE(1)
```

Comment:
I don't know which dialect is this from but based on my knowledge, none of the known dialects support REPLACE with a single argument.